README.md

```diff
@@ -20,6 +20,8 @@ The configuration object may contain:
 - `config`: configuration object that may contain the following options
   - `delimiter`: the delimiter to be used in the csv (_default_: `,`)
   - `outputFolder` (_required_): the folder used to store the exported product types and attributes
+  - `exportFormat` (_optional_): output format, supported are `csv`, `xlsx` (_default_: `xlsx`)
```

Comment:
Default should `csv` to keep consistency with other tools. However it seems that `csv` is the default if I look at [this](https://github.com/sphereio/sphere-product-type-export/pull/7/files#diff-1e559ba77e003cf58c45fbd5144942f0R18). Maybe it's even [declared](https://github.com/sphereio/sphere-product-type-export/pull/7/files#diff-08d9b720e9fc0ac6dde7e04b8991cd0bR164) twice without it being needed.